C-Plus-Plus/dp/Floyd_warshall.cpp

```diff
@@ -0,0 +1,106 @@
+#include<bits/stdc++.h>
+using namespace std;
+void floyd_waeshall(int arr[][5])
+{
+    int i, j, k;
+   //Add all vertex one by one
+    for (k = 0; k < 5; k++)
+    {
+	//pick all as a source
+        for (i = 0; i < 5; i++)
+        {
+	    //pick all as a destination
+            for (j = 0; j < 5; j++)
+            {
+                if ((arr[i][k] * arr[k][j] != 0) && (i != j))
+                {
+		    //If vertex k is on the shortest path from  
+                    // i to j, then update the value of arr[i][j] 
+                    if ((arr[i][k] + arr[k][j] < arr[i][j]) || (arr[i][j] == 0))
+                    {
+                        arr[i][j] = arr[i][k] + arr[k][j];
+                    }
+                }
+            }
+        }
+    }
+   //function to print solution
+    cout<<"\nOUTPUT"<<endl;
+    for (i = 0; i < 5; i++)
+    {
+        cout<<"\nMinimum Cost for Node: "<<i<<endl;
+        for (j = 0; j < 5; j++)
+        {
+            cout<<arr[i][j]<<"\t";
+        }
+ 
+    }
+}
+//Main function begain
```

Comment:
@yashaswibiyahut Update `begain` to `began`.

Reply:
Done @iamrajiv 
kindly review it.